### ISSUE TITLE
[A11Y] - Buttons and Elements not readable / recognizeable via Screenreader

### ISSUE THREAD

**Issue by db-ot** [opencloud-eu/desktop#821]
## Steps to reproduce
1. activate Voiceover / Screenreader mac
2. open default desktop client
3. indicate via tabbing elements from top navigation: all elements recognized as one group, not as own single elements

<img width="793" height="86" alt="Image" src="https://github.com/user-attachments/assets/4774fc5a-dfa4-4d86-ab70-7747e003659b" />

4. folder overview is indicated as one list, elements of the folder name list not recognized as single element ( not indicated / readable )

<img width="777" height="427" alt="Image" src="https://github.com/user-attachments/assets/4b573cef-0ddc-46c1-aab3-bbcb045e7097" />

5. connecting option in web is not usable for blind people single elements are not recognized / indicated as own elements

<img width="476" height="259" alt="Image" src="https://github.com/user-attachments/assets/b1bb760f-0756-48ad-b24f-02c786528a9e" />

## Expected behavior
- options from navigation list should be indicated as single clickable / readable element via Voice over
- folder in list should be readable for voiceover / screenreader

## Actual behavior
- navigation is indicated as group single elements are not readable via tabbing
- foldername are empty as list as also own single element

## Setup
- mac 13.2.1
- client app 2.0.0 rc4.0
